src/main/java/at/hannibal2/skyhanni/data/HypixelData.kt

```diff
@@ -28,6 +29,7 @@ import kotlin.concurrent.thread
 class HypixelData {
     private val group = RepoPattern.group("data.hypixeldata")
     private val tabListProfilePattern by group.pattern("tablistprofile", "§e§lProfile: §r§a(?<profile>.*)")
+    private val tabListProfilePatternRift by group.pattern("tablistprofilerift", "§d§lProfile: §r§a(?<profile>.*)")
```

Comment:
```suggestion
    private val tabListProfilePattern by group.pattern("tablistprofile", "§.§lProfile: §r§a(?<profile>.*)")
```
This could be one pattern instead of two